test/Api.Test/Auth/Controllers/AccountsControllerTests.cs

```diff
@@ -472,6 +472,51 @@ public async Task PostSetPasswordAsync_WhenSettingPasswordFails_ShouldThrowBadRe
         await Assert.ThrowsAsync<BadRequestException>(() => _sut.PostSetPasswordAsync(model));
     }
 
+    [Theory]
+    [BitAutoData]
+    public async Task Delete_WhenManagedByOrganization_ShoudThrowUnauthorizedAccessException(
```

Comment:
To get this test to pass you'll need to override the return of the feature service as well.